rust/src/lib/lib.rs

```diff
@@ -13,8 +13,8 @@ pub use crate::iface::{
     Interface, InterfaceState, InterfaceType, UnknownInterface,
 };
 pub use crate::ifaces::{
-    BaseInterface, DummyInterface, EthernetConfig, EthernetInterface,
-    Interfaces, LinuxBridgeConfig, LinuxBridgeInterface,
+    BaseInterface, DesiredInterfaces, DummyInterface, EthernetConfig,
```

Comment:
DesiredInterfaces should not be public API.

Reply:
Done